control/tests/type_conversion_test.py

```diff
@@ -149,37 +135,35 @@ def test_operator_type_conversion(opname, ltype, rtype, expected, sys_dict):
 # Note: tfx = non-proper transfer function, order(num) > order(den)
 #
 
-type_list = ['ss',  'tf',  'tfx', 'frd', 'lio', 'ios', 'arr', 'flt']
+type_list = ['ss',  'tf',  'tfx', 'frd', 'ios', 'arr', 'flt']
 conversion_table = [
-    ('ss',  ['ss',  'ss',  'tf'   'frd', 'lio', 'ios', 'ss',  'ss' ]),
-    ('tf',  ['tf',  'tf',  'tf'   'frd', 'lio', 'ios', 'tf',  'tf' ]),
-    ('tfx', ['tf',  'tf',  'tf',  'frd', 'E',   'E',   'tf',  'tf' ]),
-    ('frd', ['frd', 'frd', 'frd', 'frd', 'E',   'E',   'frd', 'frd']),
-    ('lio', ['lio', 'lio', 'E',   'E',   'lio', 'ios', 'lio', 'lio']),
-    ('ios', ['ios', 'ios', 'E',   'E',   'ios', 'ios', 'ios', 'ios']),
-    ('arr', ['ss',  'tf',  'tf'   'frd', 'lio', 'ios', 'arr', 'arr']),
-    ('flt', ['ss',  'tf',  'tf'   'frd', 'lio', 'ios', 'arr', 'flt'])]
-
-@pytest.mark.skip(reason="future test; conversions not yet fully implemented")
```

Comment:
Nice, so this table works now? (except for truediv?)

Reply:
Correct.